weed/shell/command_volume_fsck.go

```diff
@@ -541,19 +543,38 @@ func (c *commandVolumeFsck) oneVolumeFileIdsSubtractFilerFileIds(dataNodeId stri
 		return
 	}
 
+	voluemAddr := pb.NewServerAddressWithGrpcPort(dataNodeId, 0)
 	if err = c.readFilerFileIdFile(volumeId, func(nId types.NeedleId, itemPath util.FullPath) {
+		inUseCount++
+		if *c.verifyNeedle {
+			if v, ok := db.Get(nId); ok && v.Size.IsValid() {
+				newSize := types.Size(16)
```

Comment:
need a constant here, and why it is 16.

Reply:
> need a constant here, and why it is 16.

@chrislusf I honestly don't know why 16 is random